WBooks/View Controllers/ViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ViewController.swift
+//  WBooks
+//
+//  Created by Guido Marucci Blas on 4/3/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+
+final class ViewController: UITabBarController {
+    
+    // MARK: - Constants
+    
+    struct Constants {
+        static let libraryName = "Library"
+        static let libraryImage = "ic_library"
+        static let librarySelectedImage = " ic_library_active"
```

Comment:
You could directly make the constant a UIImage:
`static let librarySelectedImage = UIImage(named: "ic_library_active")`
(and there was an unnecessary space in the name)
I think is more useful to have the constants as UIImage, but apart from this, if you are storing a string, don't call the variable "Image" because it's confusing, in any case it would be "ImageName"